bot/cogs/cyber.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, bot: Bot):
             r"^.*\bwhat\b.*\belite\b.*\bemail\b.*$",
             re.IGNORECASE
         )
+        self.game_level_regex = re.compile(
+            r"^(?!:level).*(le?ve?l)\b.*?(\d{1,2}).*(\d{1,2})\b.*$",
```

Comment:
You added a negative lookahead for "level".This should not be needed because `on_message` checks if a valid command has been included, and if so, ignores it. 

Is there evidence that this protection wasn't working? It's just that this method isn't great if we decide to change our prefix in future. :)

Reply:
skiros tried the old code and it didnt work

Reply:
You can add a check for the current command prefix using a format string, rather than hardcoding the command prefix.

Reply:
True! But we really ought to investigate why the current checks are failing and if it affects the other regular expressions.

Further, :l (abbreviated) also triggers the command, but trying to code that into the regular expression is still a bad idea as cases could be missed and false-positives are likely to arise too.

Reply:
the other regular expressions aren't using a negative lookahead, but it wouldn't hurt to check them again.

Reply:
Okay. I'll investigate this evening. I'm okay with this being merged before then and we can fix later.

The negative lookahead can stay until then

Reply:
@maishams I removed the extra check locally and nothing broke. When I typed `:level 1 13`, only one output showed (handled by the command). Without the colon or with text around it, also one output showed (handled by the regex as a fallback). I tested this locally on my own Discord server.

Could you check with skiros to make sure it didn't work. Maybe they just made a suggestion without quite testing it?

Reply:
oh my old code didn't assign the variables right loool im an idiot

Reply:
@maishams 
> oh my old code didn't assign the variable right

yes
> loool im an idiot

no 
<hr>
Ah alright. So if we all agree that the extra negative lookahead isn't needed, I think we should remove it to remove clutter in the code.